.pre-commit-config.yaml

```diff
@@ -13,12 +13,19 @@ repos:
         additional_dependencies:
           - types-requests
           - types-python-dateutil
+        args: [ --disallow-untyped-defs, --disallow-any-generics,
```

Comment:
I'm afraid that this is too strict, as it will test everything. Could you please create a new entry in which you can add the newly typed files?

Reply:
I separated it and restricted to current changed file only, let me know if it is not what you wanted.